rest/route/version_create.go

```diff
@@ -61,7 +61,8 @@ func (h *versionCreateHandler) Run(ctx context.Context) gimlet.Responder {
 	}
 	p := &model.Project{}
 	opts := &model.GetProjectOpts{
```

Comment:
Is it true that we might be reading from Github here? Is there a way to test this route (or trace it back)? If h.Config already contains the full yaml, then we really should juts be passing in nil opts.